docs/configuration/load-plugin.md

```diff
@@ -0,0 +1,50 @@
+How to load a plugin
+====================
+
+Before being able to use a plugin, it must be recognized and loaded by the Perses backend.
+
+When the Perses backend is starting, it will look at a specific folder that should contain any plugin archive file. This
+folder can be set using the following configuration:
+
+```yaml
+plugin:
+  archive_path: /path/to/archive/folder
+```
+
+Note: to know how to build a plugin archive, please refer to the [plugin structure documentation](../plugins/structure.md).
+
+By default, the plugin archive folder is set to `plugins-archive` or to `/etc/perses/plugins-archive` if it's running in
+a container.
+
+Perses will extract every archive contained in this folder and will put the data into another folder. This folder can be
+set using the following configuration:
+
+```yaml
+plugin:
+  path: /path/to/plugin/folder
+```
+
+By default, the plugin folder is set to `plugins` or to `/etc/perses/plugins` if it's running in a container.
```

Comment:
Do you think it makes to add a line when Perses runs in a k8s pod? 
@Nexucis

Reply:
well in a k8s pod, Perses is still running in a container, so that should be fine I guess